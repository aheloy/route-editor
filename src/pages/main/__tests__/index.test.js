import React from 'react';

import MainPage from '../';


test('render a MainPage', () => {
  const wrapper = shallow(
    <MainPage />
  );
  expect(wrapper).toMatchSnapshot();
});
