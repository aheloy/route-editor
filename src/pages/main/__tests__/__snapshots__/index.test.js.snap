// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a MainPage 1`] = `<RouteEditor />`;
